draft-ietf-tls-ctls.md

```diff
@@ -546,6 +572,9 @@ requires some analysis, especially as it looks like a potential source
 of identity misbinding. This is, however, entirely separable
 from the rest of the specification.
 
+The idea of hashing the ephemeral public keys and their use as random 
+values was introduced by Karthik Bhargavan and requires analysis. 
+
```

Comment:
This got removed, so you should remove it, right?

Reply:
I created a note in form of an open issue.